blocks/hooks/align.js

```diff
@@ -24,7 +24,7 @@ import { getBlockSupport, hasBlockSupport } from '../api';
  * @return {Object}          Filtered block settings
  */
 export function addAttribute( settings ) {
-	if ( hasBlockSupport( settings, 'align' ) ) {
+	if ( hasBlockSupport( settings, 'align' ) && ! has( settings, [ 'attributes', 'align' ] ) ) {
```

Comment:
It would be nice to avoid this workaround. I added to make sure that tests pass, which gave me an impression that it might trigger some warnings in the browser when loading old posts.

Reply:
The failing tests I'm seeing when removing this workaround seem legitimately concerning though? 

```
Expected:
    
    <blockquote class=\"wp-block-pullquote alignundefined\"
```

Why would `alignundefined` be expected?

Worth looking into...

Reply:
Some of the failures could also be cleared up by running `npm run fixtures:regenerate` (specifically, the `_domReact` key changes)